target/linux/bmips/dts/bcm6362-netgear-dgnd3700-v2.dts

```diff
@@ -0,0 +1,225 @@
+
+	aliases {
+		led-boot = &led_power_green;
+		led-ethernet = &led_ethernet_green;
+		led-failsafe = &led_power_green;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_green;
+		led-usb = &led_usb1_green;
+		led-usb2 = &led_usb2_green;
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x00000000 0x04000000>;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&pinctrl 24 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
+		};
+
+		wlan {
+			label = "wlan";
+			gpios = <&pinctrl 25 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WLAN>;
+			debounce-interval = <60>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&pinctrl 26 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led@28 {
+			label = "green:dsl";
+			gpios = <&pinctrl 28 GPIO_ACTIVE_LOW>;
+		};
+
+		led@34 {
+			label = "red:power";
+			gpios = <&pinctrl 34 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&ehci {
+	status = "okay";
+};
+
+&leds {
+	status = "okay";
+
+	brcm,serial-leds;
+	brcm,serial-dat-low;
+	brcm,serial-shift-inv;
+	brcm,serial-mux;
+
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_leds &pinctrl_serial_led>;
+
+	led@1 {
+		reg = <1>;
+		active-low;
+		label = "green:internet";
+	};
+
+	led_power_green: led@8 {
+		reg = <8>;
+		label = "green:power";
+	};
+
+	led@9 {
+		reg = <9>;
+		active-low;
+		label = "green:wps";
+	};
+
+	led@10 {
+		reg = <10>;
+		active-low;
+		label = "green:usb1";
+	};
+
+	led@11 {
+		reg = <11>;
+		active-low;
+		label = "green:usb2";
+	};
+
+	led@12 {
+		reg = <12>;
+		active-low;
+		label = "amber:internet";
+	};
+
+	led_ethernet_green: led@13 {
+		reg = <13>;
+		active-low;
+		label = "green:ethernet";
+	};
+
+	led@14 {
+		reg = <14>;
+		active-low;
+		label = "amber:dsl";
+	};
+
+	led_usb1_green: led@16 {
+		reg = <16>;
+		active-low;
+		label = "amber:usb1";
+	};
+
+	led_usb2_green: led@17 {
+		reg = <17>;
+		active-low;
+		label = "amber:usb2";
+	};
+
+	led@18 {
+		reg = <18>;
+		active-low;
+		label = "amber:ethernet";
+	};
+};
+
+&nflash {
+	status = "okay";
+
+	nandcs@0 {
+		compatible = "brcm,nandcs";
+		#size-cells = <1>;
+		#address-cells = <1>;
+		reg = <0>;
+		nand-ecc-step-size = <512>;
+		nand-ecc-strength = <15>;
+		nand-on-flash-bbt;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			cferom: partition@0 {
+				label = "cferom";
+				reg = <0x0000000 0x0004000>;
+				read-only;
+			};
+
+			partition@4000 {
+				compatible = "brcm,wfi";
+				label = "wfi";
+				reg = <0x0004000 0x1c7c000>;
+			};
+
+			partition@1c80000 {
+				label = "flag";
+				reg = <0x1c80000 0x0040000>;
+				read-only;
+			};
+
+			partition@1cc0000 {
+				label = "pcbasn";
+				reg = <0x1cc0000 0x0040000>;
+				read-only;
+			};
+
+			partition@1d00000 {
+				label = "xxx";
```

Comment:
That's for porn?

Reply:
Porn from OEM firmware :P
See https://openwrt.org/toh/netgear/dgnd3700v2#oem_bootlog
```
Creating 14 MTD partitions on "brcmnand.0":
0x000000080000-0x000001b40000 : "rootfs"
0x000000000000-0x000000004000 : "nvram"
0x000001b40000-0x000001b80000 : "language_DEU"
0x000001dc0000-0x000001ec0000 : "scnvram"
0x000001ec0000-0x000001f00000 : "factory"
0x000000080000-0x000001cc0000 : "upgrade"
0x000001c80000-0x000001cc0000 : "flag"
0x000001cc0000-0x000001d00000 : "pcbasn"
0x000001d00000-0x000001d80000 : "xxx"
0x000001b80000-0x000001bc0000 : "language_PTB"
0x000001bc0000-0x000001c00000 : "language_RUS"
0x000001c00000-0x000001c40000 : "language_CHS"
0x000001c40000-0x000001c80000 : "language_ENU"
0x000001d80000-0x000001dc0000 : "language_dev"
```

Reply:
Well, then let's keep there default-porn read-only and forget about this :-)